app/views/groups/show.html.slim

```diff
@@ -88,19 +68,19 @@
               = link_to remove_task_group_path(task: task), method: :patch, class: 'btn btn-light', style: "border-left: none; margin-right: 5px;"  do
                 i.fa-solid.fa-xmark style=("color:gray;")
       - else
-        = t('groups.show.no_tasks')
+        = t('.has_no_tasks')
 
 br
 .actions.btn-group[role="group"]
   - if can? :request_access, @group
-    = link_to t('groups.button.request'), request_access_group_path(@group), method: :post, class: 'btn btn-important'
+    = link_to t('.button.request_membership'), request_access_group_path(@group), method: :post, class: 'btn btn-important'
   - if can? :edit, @group
     = link_to edit_group_path(@group), class: 'btn btn-important' do
       i.fa-solid.fa-pen-to-square
-      =< t('groups.button.edit')
+      =< t('common.button.edit')
     = link_to group_path(@group), class: 'btn btn-important', method: :delete, data: {confirm: 'Are you sure?'} do
       i.fa-solid.fa-trash-can style=("color:#800000")
-      =< t('groups.button.delete_group')
+      =< t('.button.delete_group')
   - if can? :leave, @group
-    = link_to t('groups.button.leave'), leave_group_path(@group), method: :post, class: "leave-btn btn btn-important #{@group.last_admin?(current_user) ? 'disabled' : ''}", data: {confirm: 'Are you sure?'}
-  = link_to t('groups.button.back'), :back, class: 'btn btn-important'
+    = link_to t('.button.leave'), leave_group_path(@group), method: :post, class: "leave-btn btn btn-important #{@group.last_admin?(current_user) ? 'disabled' : ''}", data: {confirm: 'Are you sure?'}
```

Comment:
We do have a translation for the confirmation message, don't we?

Reply:
That is something I missed. I replaced all occurrences of 'Are you sure?' with the existing translation.